client/app/src/main/java/com/example/criengine/Activities/AddBookActivity.java

```diff
@@ -135,6 +126,32 @@ public void onBackPressed() {
     }
 
     /**
+     * Retrieved from:
+     * https://stackoverflow.com/questions/11740311/android-confirmation-message-for-delete
+     * @return The confirmation dialog. If user selects to delete the book, then redirect to
+     *          the main activity.
+     */
+    private AlertDialog askForImage()
```

Comment:
don't add features that can't be accessed

Reply:
Ill add the new stuff once that camera activity gets merged then. Shouldn't be to long hopefully